pkg/core/transactor/listener.go

```diff
@@ -347,12 +350,16 @@ func loadResponseFromPub(pubKey keys.PublicKey) *node.LoadResponse {
 //nolint:unused
 func (t *Transactor) launchConsensus() {
```

Comment:
launching the consensus is a side effect of calling the `Transactor`. Maybe there is a better way to prepare the consensus outside of the Transactor and pass it to the `Chain`